overview/xarray-in-45-min.ipynb

```diff
@@ -36,23 +36,13 @@
     "%config InlineBackend.figure_format='retina'"
```

Comment:
<p>The first sentence looks like a title but maybe this is a bad rendering?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>bad rendering: seems to be matching the bold title of from cell that was moved</p>